scripts/validate/validate_rest_proxy.sh

```diff
@@ -27,7 +23,7 @@ C3_ADMIN="User:controlcenterAdmin"
 CLIENT_NAME="appSA"
 CLIENT_PRINCIPAL="User:$CLIENT_NAME"
 
-docker-compose exec tools bash -c ". /tmp/helper/functions.sh ; mds_login $MDS_URL ${SUPER_USER} ${SUPER_USER_PASSWORD}"
+./scripts/helper/refresh_mds_login.sh
```

Comment:
This command won't be correct if a user runs `./validate_rest_proxy.sh` from within the `./scripts/helper` directory.